airflow/cli/commands/connection_command.py

```diff
@@ -153,3 +155,93 @@ def connections_delete(args):
             msg = '\n\tSuccessfully deleted `conn_id`={conn_id}\n'
             msg = msg.format(conn_id=deleted_conn_id)
             print(msg)
+
+
+DIS_RESTRICT = 'restrict'
+DIS_OVERWRITE = 'overwrite'
+DIS_IGNORE = 'ignore'
+CREATED = 'created'
+DISPOSITIONS = [DIS_RESTRICT, DIS_OVERWRITE, DIS_IGNORE]
+
+
+def prep_import_status_msgs(conn_status_map):
+    msg = "\n"
+    for status, conn_list in conn_status_map.items():
+        if len(conn_list) > 0:
```

Comment:
Can you reverse the condition in this if statement and use continue? This will allow us to reduce the level of indendation.

Reply:
Updated code accordingly.